### PR TITLE
Use "org.frescobaldi.Frescobaldi" as icon name

### LINKED ISSUE #1582: no application icon in Windows
This is a minor issue. I haven't checked yet but I guess it's due to the renaming of icon file to org.frescobaldi.Frescobaldi...

Schermata del 2023-04-05 10-17-57

### PR DESCRIPTION
**Pull request by jeanas**
Should fix #1582

### CODE REVIEW

File: frescobaldi_app/icons/__init__.py [hunk 1]
```diff
@@ -42,9 +42,10 @@ def get(name):
         return _cache[name]
     except KeyError:
         icon = _cache[name] = QIcon()
-        # first try SVG
-        fname = 'icons:{0}.svg'.format(name)
-        if QFile(fname).exists():
+        # first try SVG and ico
+        if QFile(fname := 'icons:{0}.svg'.format(name)).exists():
```
Comment (PeterBjuhr) [inline frescobaldi_app/icons/__init__.py:46]: suggestion: (Saving one line and avoiding some repetitious code)
```
for ext in ['svg', 'ico']:
    if QFile(fname := f'icons:{name}.{ext}').exists():
        icon.addFile(fname)
```

Comment (PeterBjuhr) [reply frescobaldi_app/icons/__init__.py:46]: Sorry, maybe we want a break if the svg extension works - and then the line count will be the same!? :slightly_smiling_face:

**Review by PeterBjuhr — APPROVED** [review]

**Comment by 19joho66** [comment]
No, the icon was ok before and is also ok now

**Comment by fedelibre** [comment]
> I borrowed a Windows machine to test this. However, #1582 doesn't reproduce in the first place. This was the result, both without and with this patch

The screenshot seems to be run from the installer (no git menu), so it's 3.3.0 I guess.
How did you test this patch? Did you run it from source?

I cannot run the current master from source on Windows. Wait, I can if I merge #1600 into master... and now the Frescobaldi icon appears in the applications bottom bar and in the top bar. So this patch seems to solve the problem for me.
In the same Windows computer the Frescobaldi installed from the 3.3.0 installer doesn't display any icon.

**Pull request merged**